crates/ruff_cli/src/args.rs

```diff
@@ -76,11 +76,14 @@ pub enum Command {
 pub struct CheckCommand {
     /// List of files or directories to check.
     pub files: Vec<PathBuf>,
-    /// Attempt to automatically fix lint violations.
-    /// Use `--no-fix` to disable.
+    /// Apply automatic fixes to resolve lint violations.
```

Comment:
Personal preference:

```suggestion
    /// Apply fixes to resolve lint violations.
```